tests/run-test/extensions-nonexistent.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+phpt EXTENSIONS directive - extension not found
+--FILE--
+<?php
+
+require __DIR__ . '/self-invoke.php.inc';
```

Comment:
These are usually just `.inc`, not `.php.inc`.

Reply:
Resolved by not having this file anymore:P